kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -241,7 +243,11 @@ public TypeDeserializer buildTypeDeserializer(DeserializationConfig config, Java
                             return null;
                         }
 
-                        TypeIdResolver idRes = idResolver(config, baseType, subtypes, false, true);
+                        PolymorphicTypeValidator validator = BasicPolymorphicTypeValidator
+                                .builder()
+                                .allowIfSubType(baseType.getRawClass())
+                                .build();
+                        TypeIdResolver idRes = idResolver(config, baseType, validator, subtypes, false, true);
```

Comment:
There is not defined method 
https://github.com/kiegroup/droolsjbpm-integration/blob/0f831c3c0fd5f48ea22243d72cf300668f87ac29/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java#L250
@cuijulian I think there is still not correct version of jackson-databind 2.9.x instead of 2.10.x